system-tests/teamscale-properties-test/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+import kotlin.io.path.writeText
+
+plugins {
+	com.teamscale.`system-test-convention`
+}
+
+tasks.test {
+	/** These ports must match what is configured in the SystemTest class. */
+	val agentPort = 63301
```

Comment:
This agent and Teamscale port is already used by HttpRedirectSystemTest. Please pick a different one to not cause issues when executing the tests in parallel

Reply:
good point, I wasn't aware that we need to avoid conflicts here. I augmented the system test convention so ports are now picked automatically so they don't conflict. I hope I got this right ;-)